base/docs/basedocs.jl

```diff
@@ -789,6 +789,38 @@ run(pipeline(`cat test.txt`, devnull))
 """
 devnull
 
+""" 
+    `$`
+
+A `$` before a variable interpolates values into string literals, quoted expressions, and commands ([`Cmd`](@ref)). Note
```

Comment:
I am pretty sure that this will fail since `$` not escaped anymore. And the backticks (`) can be removed in the signature part. It should look like:

```
"""
    \$

A `\$` before...
"""
```